index.js

```diff
@@ -55,10 +51,11 @@ module.exports = fp(function from (fastify, opts, next) {
         opts.contentType = 'application/json'
       }
 
-      headers = Object.assign(headers, {
+      headers = {
+        ...headers,
```

Comment:
As far as I know use a plain old `for` loop is way faster than the spread operator to copy object keys and values.
Since this is a hot path you should probably use the `for` loop.
cc @mcollina

Reply:
I will change it to:
```javascript
headers['content-length'] = Buffer.byteLength(body)
headers['content-type'] = opts.contentType
```

Reply:
Why not just copy the actual headers?

Reply:
Actual headers are already copied. So append is enough.